docs/examples/Example_data_fusion_from_sensors.py

```diff
@@ -0,0 +1,506 @@
+from stonesoup.reader.base import DetectionReader
+
+
+# Create a dummy detector to parse the detections
+class DummyDetector(DetectionReader):
+    def __init__(self, *args, **kwargs):
+        self.current = kwargs['current']
+
+    @BufferedGenerator.generator_method
+    def detections_gen(self):
+        yield self.current
+
+
+# Instantiate the Kalman Tracker, without
+# specifying the detector
+UKF_tracker = MultiTargetTracker(
+    initiator=UKF_initiator,
+    deleter=deleter,
+    data_associator=data_associator_UKF,
+    updater=UKF_updater,
+    detector=None)
+
+EKF_tracker = MultiTargetTracker(
+    initiator=EKF_initiator,
+    deleter=deleter,
+    data_associator=data_associator_EKF,
+    updater=EKF_updater,
+    detector=None)
+
+# Instantiate the Particle filter as well
+PF_tracker = MultiTargetTracker(
+    initiator=PF_initiator,
+    deleter=deleter,
+    data_associator=data_associator_PF,
+    updater=PF_updater,
+    detector=None)
+
+# %%
+# 3) Run the measurement fusion algorithm and the trackers
+# --------------------------------------------------------
+# We have instantiated all the relevant components for the two
+# filters, and now we can run the simulation to generate the
+# various detections, clutter and track associations.
+# The final tracks will be passed onto a metric generator
+# plotter to measure the track accuracy.
+# We start composing the various metrics statistics available.
+
+# %%
+# Stone Soup plotting imports
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+# Load the plotter
+from stonesoup.plotter import Plotterly
+
+# Load the metric manager
+from stonesoup.metricgenerator.basicmetrics import BasicMetrics
+
+# %%
+# Metrics
+# ^^^^^^^
+
+# load the metrics for the Kalman and Particle filter
+basic_UKF = BasicMetrics(generator_name='Unscented Kalman Filter', tracks_key='UKF_tracks',
+                        truths_key='truths')
+basic_EKF = BasicMetrics(generator_name='Extended Kalman Filter', tracks_key='EKF_tracks',
+                         truths_key='truths')
+basic_PF = BasicMetrics(generator_name='Particle Filter', tracks_key='PF_tracks',
+                        truths_key='truths')
+
+# Load the OSPA metric managers
+from stonesoup.metricgenerator.ospametric import OSPAMetric
+ospa_UKF_truth = OSPAMetric(c=40, p=1, generator_name='OSPA_UKF_truths',
+                           tracks_key='UKF_tracks',  truths_key='truths')
+ospa_EKF_truth = OSPAMetric(c=40, p=1, generator_name='OSPA_EKF_truths',
+                            tracks_key='EKF_tracks',  truths_key='truths')
+ospa_PF_truth = OSPAMetric(c=40, p=1, generator_name='OSPA_PF_truths',
+                           tracks_key='PF_tracks',  truths_key='truths')
+
+# Define a data associator between the tracks and the truths
+from stonesoup.dataassociator.tracktotrack import TrackToTruth
+associator = TrackToTruth(association_threshold=30)
+
+from stonesoup.metricgenerator.manager import MultiManager
```

Comment:
Can these imports move to the imports section above to make it more consistent with the rest of the example - "Stone Soup plotting and metrics imports" for example.